src/QueryReflection/PdoQueryReflector.php

```diff
@@ -71,6 +71,14 @@ protected function simulateQuery(string $queryString)
                 throw new ShouldNotHappenException('Failed to get column meta for column index '.$columnIndex);
             }
 
+            /*
+             * Native type may not bo set, for example in case of JSON column.
+             * @phpstan-ignore-next-line
```

Comment:
I have not updated the ColumnMeta to represent the actual state of PDO.getColumnMeta return value as just ignoring this line was so much easier... 🤡